README.md

```diff
@@ -38,16 +38,14 @@ In addition to being an engaging education resource it has evolved into
 an *extremely powerful* and performance-ready *live coding instrument* suitable 
 for professional artists and DJs.
 
-Whilst Education is a core focus it now sits at the intersection
-between three core domains:
+While Education is the main priority, Sonic Pi aims to find a balance between
```

Comment:
'Whilst' is a matter of personal preference I think, since both 'whilst' and 'while' are valid. I'll defer to Sam again on that one, likewise with the rest of this sentence.